### PR TITLE
FY-1355 added example of server function docs for REST API

### PR DESCRIPTION
**Pull request by AlexYup**

### CODE REVIEW

File: invoicing/functions/server/slow.js [hunk 1]
```diff
@@ -31,5 +32,47 @@ exports.REST = {
                 return resolve({data: {success: true, time}});
             }, time * 1000);
         });
+    },
+
+    $slower: {
```
Comment (lportales) [inline invoicing/functions/server/slow.js:37]: What make this function exposed ? The "scope = public" ?

File: invoicing/functions/server/slow.js [hunk 2]
```diff
@@ -31,5 +32,47 @@ exports.REST = {
...
+            {
+                name: "multi",
+                type: "number",
+                description: "multiplier; defaults to 1"
+            }
+        ],
+        output: "{success: true, time: 'seconds waited'}",
+        example: "/invoicing/slow/slow?time=120  =>  {success: true, time: 120}",
+    },
+    slower(args, content, serv, role, session) {
```
Comment (lportales) [inline invoicing/functions/server/slow.js:60]: Could be better to find a example related to the sample, we can discuss about that.

**Review by lportales — COMMENTED** [review]
What make

File: invoicing/functions/server/slow.js (hunk 1 above)
Comment (AlexYup) [reply invoicing/functions/server/slow.js:37]: yes

**Pull request merged**